buttonAdd.js

```diff
@@ -0,0 +1,51 @@
+document.getElementById("button").addEventListener("click", function () {
+  document.querySelector(".bg-modal").style.display = "flex";
+});
+
+document.querySelector(".close").addEventListener("click", function () {
+  document.querySelector(".bg-modal").style.display = "none";
+});
+
+let data = [];
+const addProduct = (ev) => {
+  ev.preventDefault(); //to stop the form submitting
```

Comment:
No need for this kind of comments, it should be clear from the code :D

Reply:
I know that the teachers say that it is a good practice to add comments but in real life, it is not. The code naming of classes, methods, etc should make the code readable and easy to understand.